### DISCUSSION TITLE
How do I SET the transcode output folder? [ANSWERED]

### DISCUSSION THREAD

**Discussion by Cultura-Difusa** [Taiko2k/Tauon#805] [Q&A] [answered]
Hello there!

I think I have done in the past, but currently it has beaten me so far. I have read the manual, does not say it. I know it can be done (I think I have done it in the past)...but:

When I press (uner "settings" "transcode") - open output folder I only see the "folder" I have set up in the past. I want to change that.

Using the latest flatpak (permissions given to mock around with a couple of folders).

Please help my stupidity.

**Comment by Cultura-Difusa** [comment, marked as answer]
Actually I have found the answer myself:

I looked for the tauon "conf" file (tauon.conf, mine being a "flatpak app" is found at ~/.var/app/com.taiko...etc.../data/Tauon....etc), fetch it and edited: there is a section called "[directories]"... there one can set the desired place to send the lossy transcodings. Still I find this way of doing things anoying... Why is this no possible over the gui? I guess it is something in the works (or I am missing something?)

**Reply by Taiko2k** [reply]
Not missing something, just I assumed changing that dir isn't something people would commonly need to do.
